### ISSUE TITLE
Add an option to sign a document myself

### ISSUE THREAD

**Issue by PocketFR** [strukturag/nextcloud-certificate24#279]
Add an option to sign a document myself without having to send me an email

**Comment by mrknister188** [comment]
that is possible.
1. Right-click on the document you want to sign and then click Request Signature
2. use the add user option
3. enter your name.
4. put the signature field
5. press signature requests
6. go to certificate24
7. Sign the document yourself (you should have a signature saved in the settings

**Comment by fancycode** [comment]
Yeah, but it would be nice to have a dedicated UI for this where you don't have to navigate around too much.

**Comment by DeBaschdi** [comment]
Any news for this Feature ?
